extensions/dependencies/code_repl_prediction.html

```diff
@@ -0,0 +1,5 @@
+<script src="{{cache_slug}}/extensions/interactions/CodeRepl/CodeReplPredictionService.js"></script>
+<script src="{{cache_slug}}/templates/dev/head/domain/classifier/PythonProgramTokenizer.js"></script>
+<script src="{{cache_slug}}/templates/dev/head/domain/classifier/SVMPredictionService.js"></script>
```

Comment:
@seanlip what about this path in production. Will it work?

Reply:
Ooh. No, it won't. Maybe keep it in extensions for now. Might want to talk to @vojtechjelinek about this.

Incidentally, you could also have a frontend pattern where each predictionService self-inits by calling predictionServiceRegistry.add(self) (or similar) when it's loaded.

Reply:
So I will store them in `extensions/classifier` directory, is that fine?

Reply:
I think so. Make sure app.yaml has the right routing.

Reply:
Done